### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Xerillius**
Removed console logs from:
/routers/userRouter.js
/routers/auth-router.js

created test suite:
/routers/auth-router.spec.js
This test suite covers the following:
registering a user
handling duplicate email rejection
logging in a user
reject incorrect password
receive error when user not found

set up testing environment in knexfile.js
removed aws logs from server.js
added testing libraries to package.json

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -19,6 +19,19 @@ module.exports = {
...
+      database: 'voicer',
+      user:     'postgres',
+      password: ''
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations'
+    }
+  },
+
```
Comment (alphaseinor) [inline knexfile.js:34]: should we import the password as a dotenv?

Comment (alphaseinor) [reply knexfile.js:34]: maybe use TEST_PASSWORD

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node index.js",
     "server": "nodemon index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "cross-env DB_ENV=testing jest --watch"
   },
```
Comment (alphaseinor) [inline package.json:10]: test script added!

File: knexfile.js (hunk 1 above)
Comment (Xerillius) [reply knexfile.js:34]: It is a local database so a password isn't necessary.

**Pull request merged**